lib/va_profile/military_personnel/service.rb

```diff
@@ -52,6 +54,37 @@ def get_service_history
         handle_error(e)
       end
 
+      # GET's a user's DoD service summary from the VAProfile API
+      # If a user is not found in VAProfile, an empty DodServiceSummaryResponse with a 404 status will be returned
+      # @return [VAProfile::MilitaryPersonnel::DodServiceSummaryResponse] response wrapper around a
+      # dod_service_summary object
+      def get_dod_service_summary
+        with_monitoring do
+          edipi_present!
+
+          response = perform(:post, identity_path, VAProfile::Models::DodServiceSummary.in_json)
+
+          DodServiceSummaryResponse.from(response)
+        end
+      rescue Common::Client::Errors::ClientError => e
+        error_status = e.status
+        if error_status == 404
```

Comment:
> If a user is not found in VAProfile, an empty DodServiceSummaryResponse with a 404 status will be returned

do you also want to check for the empty response here as well? I'm just wondering if there's a possibility for a `404` response without an empty `DodServiceSummaryResponse` 🤔